projects/uitdatabank/docs/entry-api/school-events.md

```diff
@@ -0,0 +1,356 @@
+
+### School performances
+
+School performances are events of which both the date and the location is known in advance. For example, a theater performance aimed at a toddler of 3-4 years old in hetpaleis on 14/05/2023, 14:30 - 16:00 PM.
+
+* ✅ date is known in advance
+* ✅ location is known in advance
+
+Since school performances both have a date and a location, you can create them in a very similar way to regular events:
+
+* you can use a calendarType of your preference (`single`, `multiple` or `periodic`)
+* for the location you must use the URL of an existing place as `location.@id` in the `POST /events` request of the event
+
+### Guided tours
+
+Guided tours are events that have no specific date (or the date is not known in advance), but they do have a location. For example, a guided tour in the Royal Museum of Fine Arts Antwerp.
+
+* ❌ date is not known in advance
+* ✅ location is known in advance
+
+In order to create a guided school tour you must use:
+
+* calendarType `permanent`
+* the URL of an existing place as `location.@id` in the `POST /events` request of the event
+
+### Bookable events
+
+Bookable events are events that have no specific date and location (or the date and location is not known in advance). Both the date and the location are determined in mutual agreement between the organizer or artist and the consumer (school).
+
+For example, as a school you can book Stijn Meuris for a school performance at your school or local cultural centre, on a date that suits both the artist and the school.
+
+* ❌ date is not known in advance
+* ❌ location is not known in advance
+
+In order to create a bookable school event you must use
+
+* calendarType `permanent`
+* the URL of the dummy place as `location.@id` in the `POST /events` request of the bookable events
+
+**URL of the dummy place for bookable events**:
+
+* Test environment: `https://io-test.uitdatabank.be/place/3b92c85b-a923-4895-85f5-ed056dae11e2`
+* Production environment: `https://io.uitdatabank.be/place/c3f9278e-228b-4199-8f9a-b9716a17e58f`
+
+## Creating school events
+
+For the creation of school events several extra requirements apply:
+
+1. The `audienceType` must be set to `education`
+2. The event must have an `organizer` that has the `Cultuurkuur` label
+3. Specific education related `labels` are mandatory
+4. In case of a [guided tour](#guided-tours) or a [bookable event](#bookable-events): the `calendarType` must be set to `permanent`
+5. In case of a [bookable event](#bookable-events): the `dummy place for bookable events` must be used for the location
+
+### audienceType
+
+For school events you must include an extra property `audienceType` and set the value for the property to `education`.
+
+```js
+{
+  "audienceType": "education"
```

Comment:
This is not 100% correct, the `audienceType` property needs to be nested inside an `audience` property (I'm not sure why it was designed that way in the past, but I think it had something to do with schema.org examples incorrectly being followed to the letter)

Also maybe expand this example to include the other required properties, so it's clear that this needs to be set when creating an event. (You can omit some values by using `"..."`)

For example:
```
{
  "mainLanguage": "nl",
  "name": {
    "nl": "Voorbeeld van een guided tour event"
  },
  "location": {
    "@id": "... (maybe put a URL of a location on test here)"
  },
  "terms": [{"id": "... (just put any applicable eventtype id here)"}],
  "calendarType": "permanent",
  "audience": {
    "audienceType": "education"
  }
}
```

I see that other fields are also required for school events. So maybe put this example at the bottom under an "example(s)" header and include all fields at once. That will probably give the best overview of how the payload should look like.

Reply:
Very useful remark. Added 3 examples on the bottom of the page now